src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.DefaultProxyCredentials.cs

```diff
@@ -80,15 +80,9 @@ public void SetGet_Roundtrips()
         [Theory]
         [InlineData(false)]
         [InlineData(true)]
+	[ActiveIssue(25640, TestPlatforms.Windows)] // TODO It should be enabled for managed Handler on all platforms
```

Comment:
The tab/spacing doesn't look aligned with the rest of the attributes above it.

Reply:
sorry about that. it seems like editor on that system keeps sneaking tabs in. It looked OK when viewed locally.